.eleventy.js

```diff
@@ -53,6 +58,24 @@ module.exports = (config) => {
     array && sortBy(array, sortByKeys)
   ))
 
+  config.addShortcode('readFile', (filePath) => {
+    const templateDirs = getNunjucksPaths()
+    // eslint-disable-next-line no-restricted-syntax
+    for (const templateDir of templateDirs) {
+      try {
+        return readFileSync(join(templateDir, filePath), 'utf8')
+      } catch {
+        /* Try the next include directory on error */
+      }
+    }
+    throw new Error('readFile tag: unexpectedly reached end of template include directories')
+  })
+
+  config.addShortcode(
+    'renderToString',
+    (filePath) => prettier.format(nunjucksEnv.render(filePath, {}), { parser: 'html' }),
```

Comment:
The HTML generated by some of the examples is a bit messy so am using Prettier here to format it nicely.